README.md

```diff
@@ -5,6 +5,39 @@ It is entirely written in [Go](https://golang.org/) and backed by [Redis](https:
 
 Visit the [wiki](../../wiki/) for documentation.
 
+Getting Started
+-------------------------------------------------------------------------------
+
+Make sure that you have a working [Go
+environment](https://golang.org/doc/install) and that you have configured your
+[`$GOPATH`](https://golang.org/doc/code.html#Workspaces) correctly.
+
+Clone the repository or `go get` it with:
+```shell
+$ go get github.com/skroutz/downloader
+```
+
+Install `librdkafka-dev` in order to be able to build the project. For Debian
+GNU/Linux:
+```shell
+$ sudo apt install librdkafka-dev
```

Comment:
This is not yet a dependency. I think we should update readme after the new backend is available, or at least skip this part for now.

Reply:
Oh, right! I'll update the diff for now.